internal/ixgnmi/isislsdb_test.go

```diff
@@ -103,6 +103,19 @@ func TestISISLSDBFromIxia(t *testing.T) {
 										IsType:         ygot.Uint8(1),
 										LspId:          ygot.String("0123.4567.89AB.02-34"),
 										SequenceNumber: ygot.Uint32(5),
+										Tlv: map[telemetry.E_IsisLsdbTypes_ISIS_TLV_TYPE]*telemetry.NetworkInstance_Protocol_Isis_Level_Lsp_Tlv{
+											telemetry.IsisLsdbTypes_ISIS_TLV_TYPE_EXTENDED_IPV4_REACHABILITY: &telemetry.NetworkInstance_Protocol_Isis_Level_Lsp_Tlv{
+												Type: telemetry.IsisLsdbTypes_ISIS_TLV_TYPE_EXTENDED_IPV4_REACHABILITY,
+												ExtendedIpv4Reachability: &telemetry.NetworkInstance_Protocol_Isis_Level_Lsp_Tlv_ExtendedIpv4Reachability{
+													Prefix: map[string]*telemetry.NetworkInstance_Protocol_Isis_Level_Lsp_Tlv_ExtendedIpv4Reachability_Prefix{
+														"1.1.1.1": &telemetry.NetworkInstance_Protocol_Isis_Level_Lsp_Tlv_ExtendedIpv4Reachability_Prefix{
```

Comment:
For a little less noise, you don't need to repeat the value type inside a map, i.e. this line can just be:
"1.1.1.1": {